src/chains.rs

```diff
@@ -198,7 +198,7 @@ pub fn rewrite_chain(expr: &ast::Expr, context: &RewriteContext, shape: Shape) -
             if let Some(rw) = rewrite_chain_subexpr(last_subexpr, total_span, context, shape) {
                 let line_count = rw.lines().count();
                 let fits_single_line = almost_total + first_line_width(&rw) <= one_line_budget;
-                if (line_count >= 5 && fits_single_line) || extend_last_subexr {
+                if fits_single_line && (line_count >= 5 && fits_single_line || extend_last_subexr) {
```

Comment:
Should this be `fits_single_line && (line_count >= 5 || extend_last_subexr)` (I think they are equivalent)

Reply:
Yes, just a classic "Fix a typo commit introducing another typo...